library/std/src/io/mod.rs

```diff
@@ -1418,6 +1418,27 @@ pub trait Write {
     /// The default implementation calls [`write`] with either the first nonempty
     /// buffer provided, or an empty one if none exists.
     ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use std::io::IoSlice;
+    /// use std::io::prelude::*;
+    /// use std::fs::File;
+    ///
+    /// fn main() -> std::io::Result<()> {
+    ///     let mut data1 = [1; 8];
+    ///     let mut data2 = [15; 8];
+    ///     let mut io_slice1 = IoSlice::new(&mut data1);
+    ///     let mut io_slice2 = IoSlice::new(&mut data2);
```

Comment:
`bufs` doesn't have to be `mut`.

```suggestion
    ///     let io_slice1 = IoSlice::new(&mut data1);
    ///     let io_slice2 = IoSlice::new(&mut data2);
```

Reply:
Thanks!